icons/instagram.svg

```diff
@@ -0,0 +1,5 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16" fill="none">
+  <path d="M8 2.26506C9.85542 2.26506 10.1084 2.26506 10.8675 2.26506C11.5422 2.26506 11.8795 2.43374 12.1325 2.51807C12.4699 2.68675 12.7229 2.77109 12.9759 3.0241C13.2289 3.27711 13.3976 3.53012 13.4819 3.86747C13.5663 4.12048 13.6506 4.45783 13.7349 5.13253C13.7349 5.89157 13.7349 6.06024 13.7349 8C13.7349 9.93976 13.7349 10.1084 13.7349 10.8675C13.7349 11.5422 13.5663 11.8795 13.4819 12.1325C13.3133 12.4699 13.2289 12.7229 12.9759 12.9759C12.7229 13.2289 12.4699 13.3976 12.1325 13.4819C11.8795 13.5663 11.5422 13.6506 10.8675 13.7349C10.1084 13.7349 9.93976 13.7349 8 13.7349C6.06024 13.7349 5.89157 13.7349 5.13253 13.7349C4.45783 13.7349 4.12048 13.5663 3.86747 13.4819C3.53012 13.3133 3.27711 13.2289 3.0241 12.9759C2.77109 12.7229 2.60241 12.4699 2.51807 12.1325C2.43374 11.8795 2.3494 11.5422 2.26506 10.8675C2.26506 10.1084 2.26506 9.93976 2.26506 8C2.26506 6.06024 2.26506 5.89157 2.26506 5.13253C2.26506 4.45783 2.43374 4.12048 2.51807 3.86747C2.68675 3.53012 2.77109 3.27711 3.0241 3.0241C3.27711 2.77109 3.53012 2.60241 3.86747 2.51807C4.12048 2.43374 4.45783 2.3494 5.13253 2.26506C5.89157 2.26506 6.14458 2.26506 8 2.26506ZM8 1C6.06024 1 5.89157 1 5.13253 1C4.37349 1 3.86747 1.16868 3.44578 1.33735C3.0241 1.50602 2.60241 1.75904 2.18072 2.18072C1.75904 2.60241 1.59036 2.93976 1.33735 3.44578C1.16868 3.86747 1.08434 4.37349 1 5.13253C1 5.89157 1 6.14458 1 8C1 9.93976 1 10.1084 1 10.8675C1 11.6265 1.16868 12.1325 1.33735 12.5542C1.50602 12.9759 1.75904 13.3976 2.18072 13.8193C2.60241 14.241 2.93976 14.4096 3.44578 14.6627C3.86747 14.8313 4.37349 14.9157 5.13253 15C5.89157 15 6.14458 15 8 15C9.85542 15 10.1084 15 10.8675 15C11.6265 15 12.1325 14.8313 12.5542 14.6627C12.9759 14.494 13.3976 14.241 13.8193 13.8193C14.241 13.3976 14.4096 13.0602 14.6627 12.5542C14.8313 12.1325 14.9157 11.6265 15 10.8675C15 10.1084 15 9.85542 15 8C15 6.14458 15 5.89157 15 5.13253C15 4.37349 14.8313 3.86747 14.6627 3.44578C14.494 3.0241 14.241 2.60241 13.8193 2.18072C13.3976 1.75904 13.0602 1.59036 12.5542 1.33735C12.1325 1.16868 11.6265 1.08434 10.8675 1C10.1084 1 9.93976 1 8 1Z" fill="currentcolor"/>
+  <path d="M8 4.37349C5.9759 4.37349 4.37349 5.9759 4.37349 8C4.37349 10.0241 5.9759 11.6265 8 11.6265C10.0241 11.6265 11.6265 10.0241 11.6265 8C11.6265 5.9759 10.0241 4.37349 8 4.37349ZM8 10.3614C6.73494 10.3614 5.63855 9.3494 5.63855 8C5.63855 6.73494 6.6506 5.63855 8 5.63855C9.26506 5.63855 10.3614 6.6506 10.3614 8C10.3614 9.26506 9.26506 10.3614 8 10.3614Z" fill="#141414"/>
```

Comment:
Change `fill="#141414"` to `fill="currentColor"` on this line and the next one to fix the hover state